client/src/modules/app/components/home/home.component.ts

```diff
@@ -5,7 +5,10 @@ import {SyncService} from '../../../shared/services/base/sync.service';
 import {TodoListComponent} from '../todoList/todoList.component';
 import {AppStateService} from '../../../shared/services/appState.service';
 import {Subscription} from 'rxjs/Subscription';
-import {switchMap, tap} from 'rxjs/operators';
+import {
+    switchMap,
+    take
+} from 'rxjs/operators';
```

Comment:
One line please

Reply:
np, will do, we do this actually to get less problems with merges